docs/Install/Starting_SLM-UI_manually.md

```diff
@@ -0,0 +1,32 @@
+# Starting SLM-UI manually 
+
+SLM-UI containers are automatically started by the SLM-UI installer. But, if it is not running on the host, users need to manually start it using the following commands.
+
+1.  Initially, run `<swarm-learning>/slm-ui/scripts/run-postgres -pw" supersecretpassword"` \(default DB password, unless you have changed the default password of the database using external tools like pgAdmin\).
```

Comment:
This comment is Not for this file.  

Can we rename "Running_Swarm_Learning.md".  as "Running_Swarm_Learning_using_CLI.md".  There are some references to user guide here, we need to drop it.   Do a search for "user guide" "install guide" and fix them appropriately.